test/deployContracts.js

```diff
@@ -5,7 +5,7 @@ import { assert } from 'chai'
 
 describe('deployContracts:', function () {
   it('should deploy and register all contracts', async function () {
-    let paratii = new Paratii({
+    let paratii = await new Paratii({
```

Comment:
These `await`s here should not make any different - `new Paratii` does not return a promise, so `await` does not have anything to wait for.

That said, `mocha` does some voodoo with wating for async requests itself, so perhaps there is something going on that I do not understand here.